server/models/EntityModel.spec.js

```diff
@@ -0,0 +1,48 @@
+const EntityModel = require("./EntityModel");
+const {expect} = require("chai");
+const knex = require("../database/knex");
+const mockKnex = require("mock-knex");
+const tracker = mockKnex.getTracker();
+const jestExpect = require("expect");
+
+
+
+describe("EntityModel", () => {
+  let entityModel;
+
+  beforeEach(() => {
+    mockKnex.mock(knex);
+    tracker.install();
+    entityModel = new EntityModel();
+  })
+
+  afterEach(() => {
+    tracker.uninstall();
+    mockKnex.unmock(knex);
+  });
+
+  it("getEntityByWalletName", async () => {
+    tracker.on("query", (query) => {
+      expect(query.sql).match(/select.*entity.*/);
+      query.response([{
+        id:1,
+      }]);
+    });
+    const entity = await entityModel.getEntityByWalletName("Dadior");
+    expect(entity).to.be.a("object");
+  });
+
+  it("getEntityByWalletName can not find the wallet name", async () => {
+    tracker.uninstall();
```

Comment:
It is not so easy to use the mock-knex, or maybe I'm using it in a wrong way, I feeling is: this lib is not so stable and well-structured, maybe it would be more easier to use some mock lib directly like: senior, or jest mock.